osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/ProfileTypeDto.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2023 Smart Society Services B.V.
```

Comment:
This should be Copyright 2023 Alliander N.V.

Reply:
Copyright is updated.